ext/web/lib.rs

```diff
@@ -414,6 +425,7 @@ fn op_encoding_encode_into(
   out_buf[1] = boundary as u32;
 }
 
+// TODO(bartlomieju): op2 doesn't support v8::ArrayBuffer
```

Comment:
It should. Does a `&mut v8::ArrayBuffer` work?

Reply:
This should work:

```
#[op2]
fn op_transfer_arraybuffer<'a>(
  scope: &mut v8::HandleScope<'a>,
  ab: &v8::ArrayBuffer,
) -> Result<v8::Local<'a, v8::ArrayBuffer>, AnyError> {
  if !ab.is_detachable() {
    return Err(type_error("ArrayBuffer is not detachable"));
  }
  let bs = ab.get_backing_store();
  ab.detach(None);
  let ab = v8::ArrayBuffer::with_backing_store(scope, &bs);
  Ok(ab)
}
```

Reply:
This indeed works!